### ISSUE TITLE
CTRL+Click Issue and Tools Menu Issue

### ISSUE THREAD

**Issue by Galthariel** [ssborbis/ContextSearch-web-ext#701]
Hello,

CTRL+Click Issue: I have the Quick Menu setup to popup in AUTO and when I select a word, however every time I CTRL+Click a link in a webpage it opens the Quick Menu when the expected behavior should be to open the link in a second TAB. Unfortunately, I couldn't find where I can disable that option.

Tools Menu Issue: I couldn't find a way to increase the size of the Tools Menu. I did increase it to the Max (200%) but still seems a bit small to me :-)

**Comment by ssborbis** [comment]
> CTRL+Click Issue: 

Do you have any other quick menu opening options enabled? 

> Tools Menu Issue:

Where are you increasing the size to 200%? I cant recall if there's a separate option for just the tools menu size. You may need to use some CSS in CS Options -> General -> User styles

If you are using one of the "modern" themes, the default CSS scales the icons down to 75%. If you want to set them the same size as the rest of the quick menu icons, i think you can use...

```css
#toolBar .tile {
     transform:scale(1);
}
```

**Comment by Galthariel** [comment]
> > CTRL+Click Issue:
> 
> Do you have any other quick menu opening options enabled?
It's exactly as installed meaning default. I did not changed anything. See image below

> > Tools Menu Issue:
> 
> Where are you increasing the size to 200%? I cant recall if there's a separate option for just the tools menu size. You may need to use some CSS in CS Options -> General -> User styles
> 
> If you are using one of the "modern" themes, the default CSS scales the icons down to 75%. If you want to set them the same size as the rest of the quick menu icons, i think you can use...
> 
> ```css
> #toolBar .tile {
>      transform:scale(1);
> }
> ```
That did the trick for the trick. Thanks :-)

![2024-05-26_11-30-31](https://github.com/ssborbis/ContextSearch-web-ext/assets/51491805/d6c86089-d710-4cf1-a023-f70f586184f7)

**Comment by Galthariel** [comment]
I think I fixed. I just unchecked the Simple Click Option and it worked :-)

Thanks a lot :-)

**Comment by ssborbis** [comment]
> 1 - Is there a way to choose between horizontal and vertical SEPARATORS?

No, the way the layout is done won't allow for vertical separators like that.

> 2 - Is there a method by which I can select which search engines shall be exclusive to the Quick Menu, while all search engines shall remain integral to the Context Menu? The Reason is that I want to use the Quick Menu for the ones I use more often but keep all in the Context Menu for when I need eventually.

No, but I've been toying with that idea for a long time. I think the menu *should* be more customizable, like Page TIles, where you can set the menu size and only show the engines you want, not the whole list. The problem is implementing it at this point in development. It's going to take some overhaul that I haven't found the time to do yet. Eventually, this will be the case though.
